modules/storage-api/src/main/java/org/apache/ignite/internal/storage/DataStorageManager.java

```diff
@@ -95,7 +95,6 @@ public void stop() throws Exception {
      *      UnknownDataStorageConfigurationSchema#UNKNOWN_DATA_STORAGE} and there is only one engine, then it will be the default, otherwise
      *      there will be no default.
      */
-    // TODO: IGNITE-16835 Remove it.
```

Comment:
Why didn't we remove it?

Reply:
This method is still required, IGNITE-16835 is closed as `Won't Fix`

Reply:
ok